Berksfile

```diff
@@ -3,6 +3,7 @@ source 'https://supermarket.getchef.com'
 metadata
 
 cookbook 'yum', '~> 3.0'
+cookbook 'exhibitor', git: 'git@github.com:SimpleFinance/chef-exhibitor.git'
```

Comment:
I see you added a `git:` dependency instead of using a versioned dependency. Any specific reason why this is so? You may accidentally pick up breaking changes if you are always watching master on a repository.